src/modules/media/hooks/get-media-objects.ts

```diff
@@ -8,7 +8,7 @@ import { setResults } from '@shared/store/media';
 import { GetMedia, MediaSearchFilterField, MediaSearchFilters, SortObject } from '@shared/types';
 
 export function useGetMediaObjects(
-	slug: string,
+	orgId: string,
```

Comment:
no, we don't since the elasticsearch index only contains or-ids and no slugs.
If we wanted to make elasticsearch index based on slugs, we would need to create a new index everytime the slug was changes.

![image](https://user-images.githubusercontent.com/1710840/164216105-c139b47c-0971-47fb-baee-0eabc55b4520.png)